spikesorters/yass/yass.py

```diff
@@ -0,0 +1,236 @@
+import copy
+from pathlib import Path
+import os
+import numpy as np
+from numpy.lib.format import open_memmap
+import sys
+
+import spikeextractors as se
+from ..basesorter import BaseSorter
+from ..utils.shellscript import ShellScript
+from ..sorter_tools import recover_recording
+
+try:
+    import yaml
+    import yass
+    HAVE_YASS = True
+except ImportError:
+    HAVE_YASS = False
+
+
+class YassSorter(BaseSorter):
+    """
+    """
+
+    sorter_name = 'yass'
+    requires_locations = False
+
+    _default_params = {
+        }
+
+    _params_description = {
+    }
+       
+    sorter_description = """Yass description; link to biorxiv"""
+
+    installation_mesg = """\nTo use Yass run:\n
+        >>> pip install yass-algorithm
+
+        More information ...
+    """
+
+    def __init__(self, **kargs):
+        BaseSorter.__init__(self, **kargs)
+    
+        config_default_location = '/home/cat/code/spikesorters_forked/spikesorters/yass/config_default.yaml'
+        
+        with open(config_default_location) as file:
+            self.yass_params = yaml.load(file, Loader=yaml.FullLoader)
```

Comment:
These should be loaded as `_default_params`

Reply:
Yes, that's an option so that the end user can access them and pass values.

But for now, _default_params has values slightly different than config.yaml file.  I can change it shortly.